README.md

```diff
@@ -6,6 +6,37 @@ This helper script is provided to help you simplify the onboarding to the previe
 
 Please follow the [documentation](https://learn.microsoft.com/en-us/azure/azure-vmware/deploy-arc-for-azure-vmware-solution?tabs=windows) to use the script and deploy Arc for Azure VMware Solution
 
+## Using the script
+
+### On a Windows machine
+
+Open a PowerShell window and run the following command. You will need to bypass execution policy as the script is not signed.
+As with any script downloaded from the internet, do read its contents before running it.
+
+```powershell
+Set-ExecutionPolicy -Scope Process -ExecutionPolicy Bypass; .\run.ps1 -Operation onboard -FilePath <path_to_config_avs.json>
```

Comment:
are we intentionally removing command for offboard?

Reply:
I thought we are not exposing offboard command to customer except they face any issue and then we suggest them to offboard. isn't it? we don't even mention about offboard command even in public docs. lmk if I need to take action.

Reply:
Sure, that makes sense. no action needed.